webapi/Skills/ChatSkills/ChatSkill.cs

```diff
@@ -215,47 +216,82 @@ public async Task<string> ExtractAudienceAsync(SKContext context, CancellationTo
     /// <summary>
     /// Extract chat history.
     /// </summary>
-    /// <param name="context">Contains the 'tokenLimit' controlling the length of the prompt.</param>
     /// <param name="cancellationToken">The cancellation token.</param>
     [SKFunction, Description("Extract chat history")]
-    public async Task<string> ExtractChatHistoryAsync(
+    public async Task<string> ExtractChatHistoryAsStringAsync(
```

Comment:
Renamed this to `ExtractChatHistory` and the other one to `GetAllowedChatHistoryAsync`. 

I believe SK functions don't allow non-native types as parameters, so ChatHistory isn't a valid parameter type, so this wrapper is required so that `ExtractChatHistory` can be called from a rendered prompt.

Reply:
Also added clarifying comments to `ExtractChatHistory` description.